Editor/PhraseProvider.cs

```diff
@@ -597,7 +604,7 @@ private void ShowLocalLocaleMissingSection()
                                 continue;
                             }
                             UnityEngine.Localization.Locale newLocale = UnityEngine.Localization.Locale.CreateLocale(locale.code);
-                            AssetDatabase.CreateAsset(newLocale, $"{pathToSave}/{newLocale.ToString()}.asset");
+                            AssetDatabase.CreateAsset(newLocale, $"{pathToSave}/{newLocale}.asset");
```

Comment:
Can you explain this change? Was it not needed at all?

Reply:
no, that's not needed, as string interpolation implicitly does `ToString()`. VS code gives a lot of these linting hints.